client/src/services/retrieve-mentors.js

```diff
@@ -5,3 +5,10 @@ export const retrieveMentors = (apiUrl) =>
     .get(apiUrl)
     .then((response) => response.data.data.mentors)
     .catch((error) => Promise.reject(error))
+
+export const retrieveMentor = (id) => (
```

Comment:
nice work, but as mentioned above, lets just use the data fetched from the mentors url for quicker loading

Reply:
fixed